Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1048,8 +1049,22 @@ def get_new_indicators(
         new_indicators_list = list(
             filter(last_updated_filter(start_date), new_indicators_list)  # type: ignore
         )
+        return new_indicators_list
+    else:
+        updated_indicators = []
+        # For Indicators of Compromise only
+        for indicator in new_indicators_list:
+            # Check if indicator should be added no matter what
+            # E.g. it meets the `param` requirements
+            if indicator.get("mscore") >= minimum_mscore:
+                updated_indicators.append(indicator)
+            else:
+                existing_indicators = list(IndicatorsSearcher(value=indicator.get("value")))
+                if len(existing_indicators) > 0 and int(existing_indicators[0].get("total")) > 0:
+                    updated_indicators.append(indicator)
+        return updated_indicators
 
-    return new_indicators_list
+    return []
```

Comment:
This line is never used, since you have:
```python
if X:
    return something

else:
   return something else

return something  # <-- This line never runs
```